### ISSUE TITLE
Embedded data could not be deleted keep on repeating

### ISSUE THREAD

**Issue by wiselyloh** [PromptEngineer48/Ollama#11]
HI Please help on this

I deleted the everything in db/xxxxxxx index folder and replace source_documents folder with new set of files.

However, the old embedded data still popping out from the query ,

for eg. my old files were are .html, new files are .txt

but when running the query it replied answers from > source_documents/xxx.txt: and > source_documents/xxx.html:

and even I delete the data base index folder and run again, it is even getting messier,
now  duplicated > source_documents/xxx.txt: answers will come out.

How we can clean the database or embedded data properly

Thanks.

**Comment by wiselyloh** [comment]
sorry, the use case is we need to edit and optimise the data source files every time. After testing the QA prompt. So we need fresh database after changing the source files

**Comment by lalithvaka** [comment]
True, when I try with a random document after installing and running, its still showing me information from "Think and Grow rich.pdf" 

`> source_documents/030413.hill.think.and.grow.rich.pdf:
stricken by it. The habit of reading patent medicine advertisements.
THE FEAR OF LOSS OF LOVE
The original source of this inherent fear needs but little description, because it
obviously grew out of man’s polygamous habit of stealing his fellow-man’s mate,
and his habit of taking liberties with her whenever he could.`

**Comment by avi-lightboxio** [comment]
has anyone able to flush the old data and start add your own data to source_database
?
